src/crimson/osd/pg.cc

```diff
@@ -935,25 +933,66 @@ PG::do_osd_ops_execute(
       std::move(all_completed_fut)
     );
   }, OpsExecuter::osd_op_errorator::all_same_way(
-    [this, error_func_ptr, rollbacker, failure_func_ptr]
+    [this, maybe_submit_error_log=std::move(maybe_submit_error_log),
+     rollbacker, failure_func_ptr]
     (const std::error_code& e) mutable {
 
     ceph_tid_t rep_tid = shard_services.get_tid();
     return rollbacker.rollback_obc_if_modified(e).then_interruptible(
-    [error_func_ptr, e, rep_tid, failure_func_ptr] {
+    [maybe_submit_error_log=std::move(maybe_submit_error_log),
```

Comment:
This is only used here, can we just move that code inline to here rather than declaring it in a lambda far from the usage site?